app/core/cache/redis_client.py

```
@@ -0,0 +1,111 @@
+        if self._client is None:
+            try:
+                self._client = redis.from_url(
+                    REDIS_URL,
+                    decode_responses=True,
+                    socket_connect_timeout=5,
+                    socket_timeout=5
+                )
+                # Test connection
+                self._client.ping()
+                logger.info("Redis connection established")
+            except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)